data/iupac_goldbook/meta.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name: IUPAC Compendium of Chemical Terminology
+description: |-
+    The Compendium is popularly referred to as the Gold
+    Book, in recognition of the contribution of the late Victor Gold, who
+    initiated work on the first edition. It is one of the series of IUPAC
+    Colour Books on chemical nomenclature, terminology, symbols and units
+    (see the list of source documents), and collects together terminology
+    definitions from IUPAC recommendations already published in Pure and
+    Applied Chemistry and in the other Colour Books. Terminology
+    definitions published by IUPAC are drafted by international committees
+    of experts in the appropriate chemistry sub-disciplines, and ratified
+    by IUPAC's Interdivisional Committee on Terminology, Nomenclature and
+    Symbols (ICTNS). In this edition of the Compendium these IUPAC-approved
+    definitions are supplemented with some definitions from ISO and from
+    the International Vocabulary of Basic and General Terms in Metrology,
+    both these sources are recognised by IUPAC as authoritative. The result
+    is a collection of nearly 7000 terms, with authoritative definitions,
+    spanning the whole range of chemistry.
+targets:
+    - id: definition
+      description: definition of a chemistry term
+      units:
+      type: categorical
```

Comment:
shall we add a different type?

Reply:
Yes, I guess you could see it as a `categorical` but better use something like `text` or similar?

Reply:
yes, text sounds good for me, because also the metrics would be different for evaluation on this dataset

Reply:
changed